en/snap/platform/overview.md

```diff
@@ -65,57 +67,59 @@ On **Sandbox** mode:
 
-	Please note that __server key__ for sandbox/testing is different from production. You may obtain your Merchant ID & server key from Midtrans MAP (after logging in) from the following links:
-	
-	- For sandbox/testing: [here](https://dashboard.sandbox.midtrans.com/settings/config_info)
-	- For production/real transaction: [here](https://dashboard.midtrans.com/settings/config_info)
+  ?>***Note:*** __Server key__ for *Sandbox*/testing is different from production. You may obtain your *Merchant ID* and *server key* from Midtrans MAP (after logging in) from the following links:
 
-	![shopify](./../../../asset/image/shopify-5.png ':size=400')
+  - For *Sandbox*/testing: [here](https://dashboard.sandbox.midtrans.com/settings/config_info)
+  - For *Production*/real transaction: [here](https://dashboard.midtrans.com/settings/config_info)
+
+  ![shopify](./../../../asset/image/shopify-5.png ':size=400')
 
 9. Done! Now your Shopify online shop is ready to use Midtrans as payment gateway. Your customer will see Online Payment as payment method on the checkout page.
 
-	![shopify](./../../../asset/image/shopify-6.png ':size=400')
+  ![shopify](./../../../asset/image/shopify-6.png ':size=400')
 
-10. Midtrans Snap payment page will be displayed to the customer. Payment methods that are available for this integration is all payment methods that are available on Snap product, which is explained on [this page](https://midtrans.com/payments).
+10. Midtrans Snap payment page will be displayed to the customer. Payment methods that are available for this integration, are all payment methods that are available on Snap product, which is explained on [this page](https://midtrans.com/payments). The payment methods, which are displayed on the Snap payment page, are available for integration. The integration of these methods is explained in this section. 
 
-	![shopify](./../../../asset/image/shopify-7.png ':size=400')
+    ![shopify](./../../../asset/image/shopify-7.png ':size=400')
 
 With this integration, your customer will be redirected to Snap Redirect payment page. Customer payment data is safely managed by Midtrans hosted payment web page, outside of your Shopify store web domain.
 
 <hr><br><br>
 
 ### Available Payment Methods
-Payment methods that are available for this integration is all payment methods that are available on Snap product, which is explained on [this page](https://midtrans.com/payments).
+All payment methods available on Snap, explained on [this page](https://midtrans.com/payments), are the payment methods available for integration. The integration of these methods is explained in this section. 
 
 ### Specific Payment Methods
 
-You can optionally follow these steps, if **you prefer to have the payment methods displayed on your shopify checkout page** instead.
+To have the payment methods displayed on your shopify checkout page, follow the steps given below.
+
+Sample shopify checkout page is shown below.
```

Comment:
You removed important keyword which is `optionally` and `if you prefer`. This is important to indicate is optional and only if merchants prefer this method over the previous one